ByeBoo-iOS/ByeBoo-iOS/Presentation/Common/ByeBooNicknameTextField.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  NicknameTextField.swift
+//  ByeBoo-iOS
+//
+//  Created by APPLE on 7/4/25.
+//
+
+import Combine
+import SnapKit
+import Then
+import UIKit
```

Comment:
퍼스트파티와 서드파티를 구분해주면 좋을 것 같아요~
```suggestion
import Combine
import UIKit

import SnapKit
import Then
```